<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="All Test Suite" duration-ms="31832" started-at="2021-07-12T14:51:15 CST" finished-at="2021-07-12T14:51:47 CST">
    <groups>
    </groups>
    <test name="Test" duration-ms="31832" started-at="2021-07-12T14:51:15 CST" finished-at="2021-07-12T14:51:47 CST">
      <class name="TestCase.SearchTest">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:TestCase.SearchTest@2002fc1d]" name="beforeClass" is-config="true" duration-ms="7348" started-at="2021-07-12T14:51:33 CST" finished-at="2021-07-12T14:51:41 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="searchTest()[pri:0, instance:TestCase.SearchTest@2002fc1d]" name="searchTest" duration-ms="4071" started-at="2021-07-12T14:51:41 CST" finished-at="2021-07-12T14:51:45 CST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[断言失败java.lang.AssertionError: expected [百度首页] but found [百度首页1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 断言失败java.lang.AssertionError: expected [百度首页] but found [百度首页1]
at org.testng.Assert.fail(Assert.java:99)
at TestCase.SearchTest.searchTest(SearchTest.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchTest -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:TestCase.SearchTest@2002fc1d]" name="afterClass" is-config="true" duration-ms="1553" started-at="2021-07-12T14:51:45 CST" finished-at="2021-07-12T14:51:47 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- TestCase.SearchTest -->
      <class name="TestCase.LoginTest">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:TestCase.LoginTest@2df3b89c]" name="beforeClass" is-config="true" duration-ms="8822" started-at="2021-07-12T14:51:15 CST" finished-at="2021-07-12T14:51:24 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="LoginTest()[pri:0, instance:TestCase.LoginTest@2df3b89c]" name="LoginTest" duration-ms="8981" started-at="2021-07-12T14:51:24 CST" finished-at="2021-07-12T14:51:32 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:TestCase.LoginTest@2df3b89c]" name="afterClass" is-config="true" duration-ms="758" started-at="2021-07-12T14:51:33 CST" finished-at="2021-07-12T14:51:33 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- TestCase.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- All Test Suite -->
</testng-results>
